packages/minimongo/minimongo.js

```diff
@@ -221,14 +221,26 @@ LocalCollection.Cursor.prototype.fetch = function () {
  * @locus Anywhere
  * @returns {Number}
  */
-LocalCollection.Cursor.prototype.count = function () {
+LocalCollection.Cursor.prototype.count = function (applySkipLimit) {
   var self = this;
+  // Have to respect `applySkipLimit`, so we remove limit & skip if `applySkipLimit`
```

Comment:
Ah I see, you're changing the way `count()` works on the client... I'm not sure that's a great idea, or at the very least what do you suggest we do to help users upgrade if they already have lots of code on the client that calls `count()`? Maybe if the argument is undefined we can warn the user that the logic has changed, so that they're forced to pass `{applySkipLimit: true}`? Not sure